lib/potentialbuild.rb

```diff
@@ -936,7 +941,9 @@ def post_results(compiler, pending)
       end
 
       if !pending && @config.post_results_comment
-        if !@commit_sha.nil? && @repository == @config.repository
+        if !@pr_num_for_comment.nil?
```

Comment:
OK, so this section has nothing to do with the CI mechanics, just posting the GitHub comment.  If the PR number member variable is present, we should go ahead and use that, otherwise fall back to the normal mode.